homeassistant/components/unifi/__init__.py

```diff
@@ -32,6 +32,11 @@ async def async_setup_entry(hass, config_entry):
     if not await controller.async_setup():
         return False
 
+    if config_entry.unique_id is None:
```

Comment:
Maybe add a comment that this was added to the integration in 2021.3?

Reply:
True!

Reply:
```suggestion
    # Unique ID was introduced with 2021.3
    if config_entry.unique_id is None:
```